Workbooks/Virtual Machines - Performance Analysis/Performance Counters/Performance Counters.workbook

```diff
@@ -0,0 +1,1261 @@
+            "query": "print(\"ResourceName\")",
+            "crossComponentResources": [
+              "{Workspaces}"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [
+          "{Workspaces}"
+        ],
+        "parameters": [
+          {
+            "id": "aa2368fc-ad30-4608-b96a-72abf7b1e1af",
+            "version": "KqlParameterItem/1.0",
+            "name": "Counter1",
+            "type": 2,
+            "isRequired": false,
+            "query": "Perf\r\n| where TimeGenerated {TimeRange}\r\n| summarize by CounterName, ObjectName, CounterText = CounterName\r\n| order by ObjectName asc, CounterText asc\r\n| project Counter = pack('counter', CounterName, 'object', ObjectName), CounterText, group = ObjectName",
+            "crossComponentResources": [
+              "{Workspaces}"
+            ],
+            "value": "{\"counter\":\"% Processor Time\",\"object\":\"Process\"}",
+            "isHiddenWhenLocked": false,
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "6a7306ea-247f-46ca-abca-501911f9e9d3",
+            "version": "KqlParameterItem/1.0",
+            "name": "TrendAggregator1",
+            "type": 2,
+            "isRequired": true,
+            "isHiddenWhenLocked": false,
+            "jsonData": "[\r\n    { \"value\":\"Average = round(avg(CounterValue), 2)\", \"label\":\"Average\", \"selected\":false},\r\n    { \"value\":\"P5th = round(percentile(CounterValue, 5), 2)\", \"label\":\"P5th\", \"selected\":false},\r\n    { \"value\":\"P10th = round(percentile(CounterValue, 10), 2)\", \"label\":\"P10th\", \"selected\":false},\r\n    { \"value\":\"P50th = round(percentile(CounterValue, 50), 2)\", \"label\":\"P50th\", \"selected\":false},\r\n    { \"value\":\"P80th = round(percentile(CounterValue, 80), 2)\", \"label\":\"P80th\", \"selected\":false},\r\n    { \"value\":\"P90th = round(percentile(CounterValue, 90), 2)\", \"label\":\"P90th\", \"selected\":false},\r\n    { \"value\":\"P95th = round(percentile(CounterValue, 95), 2)\", \"label\":\"P95th\", \"selected\":true}    \r\n]",
+            "timeContextFromParameter": null
+          },
+          {
+            "id": "0327f26e-cdde-4b48-b512-6c35f06ad1d0",
+            "version": "KqlParameterItem/1.0",
+            "name": "TrendAggregatorOrder1",
+            "type": 1,
+            "isRequired": false,
+            "query": "print(iff('{TrendAggregator1}' contains 'P5th'  or '{TrendAggregator1}' contains 'P10th', 'asc', 'desc'))",
```

Comment:
Is it possible to do `{TrendAggregator1:label} == 'P5th'` so it's more precise

Reply:
Sure, thats reasonable.